source/agora/utils/PrettyPrinter.d

```diff
@@ -62,7 +62,8 @@ private struct AmountFmt
 @safe unittest
 {
     immutable one = Amount(1);
-    assert(format("%s", AmountFmt(Amount.MaxUnitSupply)) == "500,000,000.0000,000");
```

Comment:
So can you help me understand the existing formatting? Because I didn't notice this before and it's very confusing to me.

Why are there 4 zeros before the comma? Shouldn't it be something like:

```
500,000,000.0,000,000
```

Help me understand the logic.

Reply:
To be fair, I expected exactly what you describe when I did the PR, but somehow the results were wrong so I just went with it. Looks like someone fixed it.